specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/stable/2023-04-06/baremetalinfrastructure.json

```diff
@@ -198,6 +198,83 @@
           }
         }
       },
+      "delete": {
+        "tags": [
+          "BareMetalInfrastructure"
+        ],
+        "operationId": "AzureBareMetalInstances_Delete",
+        "summary":"Deletes an Azure BareMetal instance.",
+        "description": "Deletes an Azure BareMetal instance for the specified subscription, resource group, and instance name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureBareMetalInstanceNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
As a note to anyone else that sees this, our RP does _not_ support PUT or DELETE operations. However, as a requirement for GA, we need to return a response for them. The ARM team has gave us the greenlight for only returning a "Not supported" code with a message stating the official routes needed to perform it. Which is why we do not have any 2xx responses listed.

Second, this is not a breaking change. This api version, 2023-04-06, received an exception very recently when it was created for not adding PUT/DELETE operations. This PR simply adds a formal "Not supported" response. No actual functionality with this operation has changed.